src/System.IO.Pipelines.Networking.Libuv/Interop/SockAddr.cs

```diff
@@ -85,7 +85,7 @@ public unsafe IPEndPoint GetIPEndPoint()
             {
                 // otherwise IPv6
                 var bytes = new byte[16];
```

Comment:
Shame the `IPAddress(byte* address` overloads are internal, else could `stackalloc` this